torch/csrc/jit/codegen/cuda/kernel_cache.h

```diff
@@ -21,16 +23,63 @@ namespace cuda {
 // TODO: Figure out how we want to cache based on heuristics, should probably
 // use something similar. Heuristics may also return a LaunchParams object.
 // TODO: Validate it is included in the string.
+
 class FusionExecutorCache {
  public:
-  FusionExecutor* getExecutor() const {
-    return entry;
-  }
+  // FusionExecutorCache(Fusion* fusion, CompileOptions options);
```

Comment:
Comment plz. What is the cache for?

Reply:
Why a vector of fusion executors?

Reply:
OOps, I do have comment on cache level, but it's in the wrong place (`manager.cpp`)🤦 I'll move it over and put more inline comments in headers.